templates/search.html

```diff
@@ -164,6 +167,48 @@ <h3 class="title">
               {% endfor %}
             </div>
 
+          {% elif form.facets.kind.0 == 'News' %}
+            <div style="column-count: 2">
+              <div class="country_status">
+                  <span class="country">
+                  {% for state in res.state_name %}
+                    <img src="/assets/imgs/flag-{{ state|lower|cut:" " }}.png"
+                         alt="{{ state }} flag"
+                         height="20px">&nbsp;{{ state }}
+                {% endfor %}</span>
+              </div>
+              <img src="{{ res.image }}" style="column-span: 1">
+              <p>
+                <div>
+                  <b>Programme:</b>
+                  <ul class="search_listing">
+                    {% for programme in res.programme_name %}
+                      <li><span>{{ programme }}</span></li>
+                    {% endfor %}
+                  </ul>
+                </div>
+                {% if res.project_name %}
+                  <div><b>Project:</b> {{ res.project_name.0 }}</div>
+                  <div><b>Project status:</b> {{ res.project_status.0 }}</div>
+                  <div><b>Funded by :</b> {{ res.financial_mechanism_ss.0 }}</div>
+                  <div><b>Sector:</b>
```

Comment:
Most of the times it will be a single sector. Try not to use a bullet unless there are more